test/test_helper.rb

```diff
@@ -68,6 +68,7 @@ def run
 
     def setup
       Kubectl.any_instance.expects(:run).never if ban_net_connect? # can't use mocha in Minitest::Test#run
+      @namespace = nil
```

Comment:
This needs to go in `run` instead. Right now it is overwriting the value set in the integration test helper.